Header header
float32 lin_vel         # linear velocity in m/s
float32 phi_f           # steering angle in rad front axis
float32 phi_r           # steering angle in rad rear axis

uint8 NO_BLINK = 0
uint8 BLINK_LEFT = 1
uint8 BLINK_RIGHT = 2
uint8 BLINK_BOTH = 3
uint8 blink_com # turn indicator
